source/objects.txt

```diff
@@ -82,8 +82,8 @@ The second part, ``None``, indicates the return type of the signal,
 usually ``None``.
 
 ``(int,)`` indicates the signal arguments, here, the signal will only
-take one argument, whose type is int. This argument type list must end with a
-comma.
+take one argument, whose type is int. This argument type must be a sequence,
+so here as a tuple with one element must end with a comma.
```

Comment:
I find this sentence difficult to read, in particular the last part "must end with a comma" is kind of confusing. Maybe you can re-formulate it.

Reply:
To be honest, I thought about removing the part `must end with a comma` (it's just a Python-related knowledge when we use tuple), but I see the previous author decided it's kind of important explanation. 

What do you think about `Types of arguments required by signal are declared as a sequence, here is one-element tuple.` or maybe even just drop this sentence?